launch/tier4_perception_launch/launch/traffic_light_recognition/traffic_light_node_container.launch.py

```diff
@@ -63,11 +69,21 @@ def add_launch_arg(name: str, default_value=None, description=None):
     # traffic_light_classifier
     add_launch_arg("classifier_type", "1")
     add_launch_arg(
-        "classifier_model_path",
+        "classifier_model_path_car",
         os.path.join(classifier_share_dir, "data", "traffic_light_classifier_efficientNet_b1.onnx"),
     )
     add_launch_arg(
-        "classifier_label_path", os.path.join(classifier_share_dir, "data", "lamp_labels.txt")
+        "classifier_model_path_ped",
+        os.path.join(
+            classifier_share_dir, "data", "ped_traffic_light_classifier_efficientNet_b1.onnx"
+        ),
+    )
+    add_launch_arg(
+        "classifier_label_path_car", os.path.join(classifier_share_dir, "data", "lamp_labels.txt")
+    )
+    add_launch_arg(
+        "classifier_label_path_ped",
+        os.path.join(classifier_share_dir, "data", "lamp_labels_ped.txt"),
```

Comment:
Cloud you share `lamp_labels_ped.txt`?

Reply:
Thanks, I added it in description. https://drive.google.com/file/d/1Gn-D0vM9Wf1R3B1kwwrRPpgqJ4O-dniD/view?usp=drive_link